templatetags/htk_tags.py

```diff
@@ -285,6 +273,20 @@ def is_editable_by_context_user(context, obj):
         is_editable = False
     return is_editable
 
+@register.simple_tag(takes_context=True)
+def has_permission(context, permission_codename):
+    request = context.get('request', {}).get('request', None)
+    if request and request.user:
```

Comment:
`user` will always be there on a Django request, so we don't need to check for this

It's whether the user is `user.is_authenticated` or not

Reply:
I see, I made the assumption that this is behind admintools door, but this tag can be used anywhere. So we only want to do this when user is authenticated.

Reply:
Django `request.user` is always an object. it can be an `AnonymousUser`

Reply:
https://docs.djangoproject.com/en/1.11/ref/contrib/auth/#anonymoususer-object